server/models/timeModel.js

```diff
@@ -0,0 +1,43 @@
+class Time{
```

Comment:
You can't reinvent the wheel. If you want to do something like this you must use dayjs.
Call this WeekPhaseLimit or WeekPhaseBoundary or make a model with both the startTime and endTime and call it WeekPhaseInterval (better choice).

Reply:
The problem with jdays was that I can't define "Monday at 15:00", but I have to define "2021, November, 28, Monday at 15:00". So I thought I had 3 choices. 1) I use jdays defining a random yyyy/mm/dd and ignoring those fields in my code. 2) I find an alternative to jdays that doesn't require me to define a random values for yyyy/mm/dd. 3) I write the class with only the info needed. If 1) is considered a better choice, I'll do that.

Reply:
You can create the object WeekPageInterval, which should contain:
```
- startWeekday //0,1,2,3,4,5,6
- startHour // 12
- startMinute // 30

- endWeekday
- endHour
- endMinute

```
And then define a function to check if a dayjs date is contained in the interval:
```
function isDateWithinInterval(date):
   dateWeekday = date.weekday
   dateHour = date.hour
   dateMinute = date.minute
   
   dateStr = dateWeekday+dateHour+dateMinute

   startStr = startWeekday+startHour+startMinute
   endStr = endWeekday+endHour+endMinute

   return (dateStr >= startStr && dateStr < endStr)
```